src/Themes/VuetifyTheme/Views/Content-VListTitle.liquid

```diff
@@ -0,0 +1 @@
+<v-subheader>{{ Model.ContentItem.Content.VListTitle.Title.Text | raw }}</v-subheader>
```

Comment:
We should rename this component to VSubheader and add the props: https://dev.vuetifyjs.com/en/api/v-subheader/#props